.gitignore

```diff
@@ -21,6 +21,7 @@ test-coverage/**
 .idea/**/tasks.xml
 .idea/**/usage.statistics.xml
 .idea/**/shelf
+.idea/*.xml
```

Comment:
This is xml rather than iml and includes many files which are checked in. This line needs to be removed.

Reply:
This is not a mistake it’s on purpose. There are xml files which get added due to plugins or new versions for example encodings.xml, hydra.xml. These files should never be committed but cause issues during rebasing. So, having this line is correct

Reply:
This is not a mistake it’s on purpose. There are xml files which get added due to plugins or new versions for example encodings.xml, hydra.xml. These files should never be committed but cause issues during rebasing. So, having this line is correct

Reply:
It's too broad. There are 2 or 3 xml files which can be ignored and a bunch which must be committed. Unfortunately, encodings.xml is one that is probably required by TeamCity. I'll test that after I test removing the imls.

Reply:
Sorry for being dense here. In none of the other projects I have worked with to get things to work correctly, was there a necessity for the xml files. If we want an off cycle change we can always work around it. I would rather be too broad than too restrictive

Reply:
Sorry for being dense here. In none of the other projects I have worked with to get things to work correctly, was there a necessity for the xml files. If we want an off cycle change we can always work around it. I would rather be too broad than too restrictive